apps/backoffice-v2/src/pages/Entity/components/EditableDetails/EditableDetails.tsx

```diff
@@ -156,6 +158,25 @@ export const EditableDetails: FunctionComponent<IEditableDetails> = ({
                       !Array.isArray(value),
                     ].every(Boolean);
                     const isSelect = isInput && !!dropdownOptions;
+                    const inputType = (() => {
+                      if (format) {
+                        return format;
+                      }
+
+                      if (type === 'string') {
+                        return 'text';
+                      }
+
+                      if (isValidDate(value, false) || isValidIsoDate(value) || type === 'date') {
```

Comment:
`isValidDate({ value, isStrict: false })`

Reply:
https://medium.com/@amlcurran/clean-code-the-curse-of-a-boolean-parameter-c237a830b7a3